app/src/androidTest/java/campus/tech/kakao/map/KakaoMapActivityTest.kt

```diff
@@ -0,0 +1,67 @@
+package campus.tech.kakao.map
+
+import android.content.Intent
+import androidx.test.core.app.ActivityScenario
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import campus.tech.kakao.map.model.search.Place
+import campus.tech.kakao.map.view.ActivityKeys
+import campus.tech.kakao.map.view.kakaomap.KakaoMapActivity
+import org.hamcrest.Matchers.not
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+private const val TEST_PLACE_NAME = "Test Place"
+private const val TEST_CATEGORY_NAME = "Test Category"
+private const val TEST_ADDRESS_NAME = "Test Address"
+private const val TEST_X = "36.37003"
+private const val TEST_Y = "127.34594"
+
+@RunWith(AndroidJUnit4::class)
+class KakaoMapActivityTest {
+
+    @Rule
```

Comment:
코틀린에서는 아래와 같이 룰을 선언해주어야 합니다. 
```suggestion
    @get:Rule
```